cloud/cloudlaunch/cloudlaunch.go

```diff
@@ -165,10 +163,6 @@ func (c *Config) MaybeDeploy() {
 	}
```

Comment:
@ChrisMcKenzie, you missed the text in `doLaunch` flag above:

```
doLaunch = flag.Bool("cloudlaunch", false, "Deploy or update this binary to the cloud. Must be on Linux, for now.")
```

You should remove the "Must be on Linux, for now." sentence.